File: app/src/main/java/ywdemo/example/yaoxiaowen/until/LogUtil.java
package ywdemo.example.yaoxiaowen.until;

import android.util.Log;

public class LogUtil {

    private static final String PREFIX = "LogPrefix";

    public static void v(String tag, String msg) {
        Log.v(PREFIX + " " + tag, msg);
    }

    public static void d(String tag, String msg) {
        Log.d(PREFIX + " " + tag, msg);
    }

    public static void i(String tag, String msg) {
        Log.i(PREFIX + " " + tag, msg);
    }

    public static void iWithThreadInfo(String tag, String msg) {
        // 已删除 ThreadUtil 引用，使用 Thread.currentThread() 替代
        Log.i(PREFIX + " " + tag, msg + ", " + Thread.currentThread().getName());
    }

    public static void w(String tag, String msg) {
        Log.w(PREFIX + " " + tag, msg);
    }

    public static void e(String tag, String msg) {
        Log.e(PREFIX + " " + tag, msg);
    }

    public static void v(String msg) {
        Log.v(PREFIX, msg);
    }

    public static void d(String msg) {
        Log.d(PREFIX + " ", msg);
    }

    public static void i(String msg) {
        // 已删除 ThreadUtil 引用，使用 Thread.currentThread() 替代
        Log.i(PREFIX + " ", msg + ", " + Thread.currentThread().getName());
    }

    public static void iWithThreadInfo(String msg) {
        Log.i(PREFIX + " ", msg);
    }

    public static void w(String msg) {
        Log.w(PREFIX + " ", msg);
    }

    public static void e(String msg) {
        Log.e(PREFIX, msg);
    }
}

File: app/src/main/java/ywdemo/example/yaoxiaowen/until/LogUtil.kt
package ywdemo.example.yaoxiaowen.until

import android.util.Log

class LogUtil  {

    companion object {

        private const val  prefix = "LogPrefix"

        @JvmStatic
        fun v(tag: String?, msg: String) {
            Log.v("$prefix $tag", msg);
        }

        @JvmStatic
        fun d(tag: String?, msg: String) {
            Log.d("$prefix $tag", msg);
        }

        @JvmStatic
        fun i(tag: String?, msg: String) {
            Log.i("$prefix $tag", msg);
        }

        @JvmStatic
        fun w(tag: String?, msg: String) {
            Log.w("$prefix $tag", msg);
        }

        @JvmStatic
        fun e(tag: String?, msg: String) {
            Log.e("$prefix $tag", msg);
        }



        @JvmStatic
        fun v( msg: String) {
            Log.v("$prefix", msg);
        }

        @JvmStatic
        fun d( msg: String) {
            Log.d("$prefix ", msg);
        }

        @JvmStatic
        fun i( msg: String) {
            Log.i("$prefix ", msg);
        }

        @JvmStatic
        fun w( msg: String) {
            Log.w("$prefix ", msg);
        }

        @JvmStatic
        fun e( msg: String) {
            Log.e("$prefix", msg);
        }
    }


}